### DISCUSSION TITLE
Can you please link visual testing differnce html to report html

### DISCUSSION THREAD

**Discussion by radhakrishnanakireddy** [seleniumbase/SeleniumBase#1168] [Ideas]
Hi,
Thanks a lot for implelemting visual differnce html report for visual testing, but it is getting created in lastestlogs folders
Can we have feature that like to have viusal differnce html page in Report html. why because it will helpful when we generate reports html for visual testing.

Please let me know your views on it.

Thank you,
RK

**Comment by mdmintz** [comment]
Hi @radhakrishnanakireddy, That's one of the reasons for the new Dashboard: A place to put additional logs/data for tests because everything can't fit on the html report without breaking alignment, etc.
Running with both ``--dashboard`` and ``--html=report.html`` will generate both. There are several log files that are created with visual testing logs, such as screenshots of before & after, as well as the side_by_side.html file which includes src links to both of them. It would take up considerable extra space in the assets/ folder, which is where html report data gets saved to for that. Here's where the link is in the ``dashboard.html`` file:

<img width="500" alt="Screen Shot 2022-01-10 at 8 54 07 AM" src="https://user-images.githubusercontent.com/6788579/148778475-a244c3ea-3b39-4846-b7b6-e9aeb2197d93.png">

<img width="400" alt="Screen Shot 2022-01-10 at 9 06 55 AM" src="https://user-images.githubusercontent.com/6788579/148779317-147a5f83-2046-46a7-8ad6-680425a4ae9e.png">

**Reply by radhakrishnanakireddy** [reply]
Thanks a lot for the reply. I got it!
